webview/src/setup/components/App.test.tsx

```diff
@@ -14,21 +14,33 @@ jest.mock('../../shared/components/codeSlider/CodeSlider')
 const { postMessage } = vsCodeApi
 const mockPostMessage = jest.mocked(postMessage)
 
-const setData = (
-  cliCompatible: boolean | undefined,
-  hasData: boolean | undefined,
-  isPythonExtensionInstalled: boolean,
-  projectInitialized: boolean,
+const setData = ({
+  canGitInitialize,
+  cliCompatible,
+  hasData,
+  isPythonExtensionInstalled,
+  needsGitInitialized,
+  projectInitialized,
+  pythonBinPath
+}: {
+  canGitInitialize: boolean | undefined
+  cliCompatible: boolean | undefined
+  hasData: boolean | undefined
+  isPythonExtensionInstalled: boolean
+  needsGitInitialized: boolean | undefined
+  projectInitialized: boolean
   pythonBinPath: string | undefined
-) => {
+}) => {
```

Comment:
[F] I changed the signature to a destructured object because it was getting out of control.